src/main/resources/templates/tableHeader.vm

```diff
@@ -6,7 +9,14 @@
     <th></th>
 </tr>
 <tr>
-    <th id="stats-header-feature">Feature</th>
+    #if($tagPage)
+        <th id="stats-header-tag">Tag</th>
```

Comment:
That unfortunatly would't work since it is set to tag/feature name (https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/resources/templates/pages/tagReport.vm#L15), so I made a variable table_key and set it in templates instead.